zipkin-lens/package.json

```diff
@@ -15,12 +15,12 @@
     "@fortawesome/free-brands-svg-icons": "6.5.1",
     "@fortawesome/free-solid-svg-icons": "6.5.1",
     "@fortawesome/react-fontawesome": "0.2.0",
-    "@lingui/cli": "3.0.0-13",
-    "@lingui/core": "3.0.0-13",
-    "@lingui/macro": "3.0.0-13",
-    "@lingui/react": "3.0.0-13",
+    "@lingui/cli": "4.6.0",
+    "@lingui/core": "4.6.0",
+    "@lingui/macro": "4.6.0",
+    "@lingui/react": "4.6.0",
     "@material-ui/core": "^4.1.1",
-    "@material-ui/data-grid": "4.0.0-alpha.37",
+    "@material-ui/data-grid": "4.0.0-alpha.24",
```

Comment:
Temporarily downgrade to pass `npm install`.
`@material-ui/data-grid 4.0.0-alpha.37`'s peer dependency is react 17. But we are still using react 16.
I'll update this again after update react version.